health.go

```diff
@@ -19,13 +19,13 @@ package marathon
 // HealthCheck is the definition for an application health check
 type HealthCheck struct {
 	Command                *Command `json:"command,omitempty"`
-	Protocol               string   `json:"protocol,omitempty"`
-	Path                   string   `json:"path,omitempty"`
-	GracePeriodSeconds     int      `json:"gracePeriodSeconds,omitempty"`
-	IntervalSeconds        int      `json:"intervalSeconds,omitempty"`
-	PortIndex              int      `json:"portIndex,omitempty"`
+	Protocol               string   `json:"protocol,"`
```

Comment:
Removing `omitempty` means we cannot let the client rely on Marathon's default settings anymore. Therefore, I would pointerize the fields which have zero value semantics (namely, `GracePeriodSeconds` and `PortIndex`), provide convenience setters, and leave the rest untouched.

In case we decide to stick to the current approach: `Protocol` has a trailing comma after the tag specifiers.